src/main/java/christmas/domain/eventplanner/strategy/EventStrategy.java

```diff
@@ -0,0 +1,23 @@
+package christmas.domain.eventplanner.strategy;
+
+import christmas.domain.customer.Customer;
+import christmas.domain.restaurant.Gift;
+
+public interface EventStrategy {
+
+    boolean checkEligibility(Customer customer);
+
+    int calculateDiscount(Customer customer);
+
+    default int calculateBenefit(Customer customer) {
+        return calculateDiscount(customer);
+    }
```

Comment:
```suggestion
    int calculateBenefit(Customer customer);
```
차라리 Discount가 default method인 인터페이스를 하나 추가하는 방식으로 속성을 만들었다면 어땠을까요?
calculateBenefit 메서드는 모든 클래스에서 구현이 필요하니까요. 

```java
public interface Discount {
    default int calculateDiscount(Customer customer) {
        return calculateBenefit(customer);
    }
}

public class SpecialEventStrategy implements EventStrategy, Discount {...}
```